incompatible-pointer-types error while building with conan

Hello I hope you are doing well. I'm trying to use openblas with conan but the compilation fails with: 
lapack-netlib/SRC/spstf2.c:796:35: error: passing argument 1 of ‘dmaxloc_’ from incompatible pointer type [-Wincompatible-pointer-types]
   itemp = mymaxloc_(&work[1], &i__2, &i__3, &c__1); (converting a float* to a double*)
The same error is also in:
lapack-netlib/SRC/spstrf.c

I'm compiling with gcc14. 

Thanks for your time and your work.

--- a/lapack-netlib/SRC/cpstf2.c
+++ b/lapack-netlib/SRC/cpstf2.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 

--- a/lapack-netlib/SRC/cpstrf.c
+++ b/lapack-netlib/SRC/cpstrf.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 

--- a/lapack-netlib/SRC/spstf2.c
+++ b/lapack-netlib/SRC/spstf2.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 

--- a/lapack-netlib/SRC/spstrf.c
+++ b/lapack-netlib/SRC/spstrf.c
@@ -256,7 +256,7 @@ static char junk[] = "\n@(#)LIBF77 VERSION 19990503\n";
 #define myceiling_(w) {ceil(w)}
 #define myhuge_(w) {HUGE_VAL}
 //#define mymaxloc_(w,s,e,n) {if (sizeof(*(w)) == sizeof(double)) dmaxloc_((w),*(s),*(e),n); else dmaxloc_((w),*(s),*(e),n);}
-#define mymaxloc_(w,s,e,n) dmaxloc_(w,*(s),*(e),n)
+#define mymaxloc_(w,s,e,n) smaxloc_(w,*(s),*(e),n)
 
 /* procedure parameter types for -A and -C++ */
 

